plugins/inputs/modbus/configuration_request.go

```diff
@@ -269,13 +269,13 @@ func (c *ConfigurationPerRequest) fieldID(seed maphash.Seed, def requestDefiniti
 	if err := mh.WriteByte(0); err != nil {
 		return 0, err
 	}
-	if _, err := mh.WriteString(def.Measurement); err != nil {
+	if _, err := mh.WriteString(field.Measurement); err != nil {
```

Comment:
Not using the global measurement name shouldn't cause any backward compatibility issues?

Reply:
In the caller function, the field measurement is set to `def.Measurement` [by default](https://github.com/influxdata/telegraf/blob/a4956057b3d3c9414f2d23339b09e2baf7c3c07f/plugins/inputs/modbus/configuration_request.go#L107). So if the field _does not_ overwrite the measurement, the code is equivalent.